packages/web-react/src/components/Header/README.md

```diff
@@ -4,398 +4,541 @@ The Header is a highly variable and customizable component. It comes in several
+| `UNSAFE_style`     | `CSSProperties` | —       | ✕        | Custom style      |
 
-The rest of the properties are based on the type of `<ul>`. [Docs][DialogUListDocs]
+The component further inherits properties from the [`<li>`][mdn-li-element] element.
 
-## HeaderDialogNavItem
+##### HeaderLink API
 
-### API
+| Name               | Type            | Default | Required | Description          |
+| ------------------ | --------------- | ------- | -------- | -------------------- |
+| `children`         | `ReactNode`     | —       | ✕        | Children node        |
+| `isCurrent`        | `bool`          | `false` | ✕        | Mark link as current |
+| `UNSAFE_className` | `string`        | —       | ✕        | Custom class name    |
+| `UNSAFE_style`     | `CSSProperties` | —       | ✕        | Custom style         |
 
-| Name               | Type            | Default | Required | Description                           |
-| ------------------ | --------------- | ------- | -------- | ------------------------------------- |
-| `children`         | `ReactNode`     | —       | ✕        | Children node                         |
-| `UNSAFE_className` | `string`        | —       | ✕        | HeaderDialogNavItem custom class name |
-| `UNSAFE_style`     | `CSSProperties` | —       | ✕        | HeaderDialogNavItem custom style      |
+The component further inherits properties from the [`<a>`][mdn-a-element] element.
 
-The rest of the properties are based on the type of `<li>`. [Docs][DialogListItemDocs]
+##### HeaderButton API
 
-## HeaderDialogText
+| Name               | Type            | Default | Required | Description       |
+| ------------------ | --------------- | ------- | -------- | ----------------- |
+| `children`         | `ReactNode`     | —       | ✕        | Children node     |
+| `UNSAFE_className` | `string`        | —       | ✕        | Custom class name |
+| `UNSAFE_style`     | `CSSProperties` | —       | ✕        | Custom style      |
 
-### API
+The component further inherits properties from the [`<button>`][mdn-button-element] element.
 
-| Name               | Type            | Default | Required | Description                        |
-| ------------------ | --------------- | ------- | -------- | ---------------------------------- |
-| `children`         | `ReactNode`     | —       | ✕        | Children node                      |
-| `UNSAFE_className` | `string`        | —       | ✕        | HeaderDialogText custom class name |
-| `UNSAFE_style`     | `CSSProperties` | —       | ✕        | HeaderDialogText custom style      |
+## Header Dialog
 
-The rest of the properties are based on the type of `<span>`. [Docs][DialogSpanDocs]
+Header Dialog is Spirit's solution for responsive navigation and selected use
+cases such as the user menu. Please note Header Dialog is not intended to be
+used for second-level navigation in general.
 
-## HeaderLink
+```jsx
+const [isOpen, setOpen] = useState(false);
```

Comment:
```suggestion
const [isOpen, setOpen] = React.useState(false);
```

Maybe this is better? Because the reader will know you mean react usestate?

Reply:
OK, fixed.